docs/docsource/reference/blobserver/Account/ListContainers.rst

```diff
@@ -0,0 +1,377 @@
+.. _List Containers:
+
+List Containers
+===============
+
+The List Containers operation returns a list of the containers under the
+specified storage account.
+
+Request
+-------
+
+The List Containers request may be constructed as follows. HTTPS is
+recommended. Replace ``myaccount`` with the name of your storage account, and
+``example.com`` with your endpoint's domain name or IP address.
+
+======= ================================================= ============
+Method  Request URI                                       HTTP Version
+======= ================================================= ============
+GET     ``https://myaccount.blob.example.com/?comp=list`` HTTP/1.1
+======= ================================================= ============
+
+The URI must include the forward slash (/) to separate the host name from the
+path and query portions of the URI. For List Containers operation, the path
+portion of the URI remains empty.
+
+Emulated Storage Service Request
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When making a request against the emulated storage service, specify the emulator
+hostname and Blob service port as ``127.0.0.1:10000``, followed by the emulated
+storage account name:
+
+======= ===================================================== ============
+Method  Request URI                                           HTTP Version
+======= ===================================================== ============
+GET     ``http://127.0.0.1:10000/devstoreaccount1?comp=list`` HTTP/1.1
+======= ===================================================== ============
+
+.. note::
```

Comment:
Not sure if want to leave in the azure storage emulator stuff

Reply:
Removed all.

Reply:
Wouldn't it be better to remove the entire section titled `Emulated Storage Service Request`? lines 29 - 45

Reply:
Yes it would. I thought I got all those.